theme/templates/resource-landing-page/references.html

```diff
@@ -45,6 +45,10 @@ <h4>Related Resources</h4>
                                     This resource is described by:
                                 {% elif relation.type == "IsReferencedBy" %}
                                     This resource is referenced by:
+                                {% elif relation.type == "isVersionOf" %}
```

Comment:
If we implement a new property (display_type) for the Relation class, we don't have to do all these if...elifs in the template. We can have one line {{ relation.display_type }}. Just a thought.